lib/conjur/command/audit.rb

```diff
@@ -24,12 +24,22 @@ class << self
           "reported #{statement}"+ message_part
         }
       }
+
+      def ssh_message(e)
+        s = "#{e[:system_user]}"
+        s << " " << (e[:allowed] ? "ran" : "attempted to run")
+        s << " '" << e[:command] << "' as " << e[:target_user]
+        s
+      end
       
       def short_event_format e
         e.symbolize_keys!
         s = "[#{Time.parse(e[:timestamp])}]"
         s << " #{e[:user]}"
         s << " (as #{e[:acting_as]})" if e[:acting_as] != e[:user]
+        if e[:facility] == 'ssh' && e[:action] == 'sudo'
+          e[:audit_message] = ssh_message(e)
```

Comment:
Given the condition above I'd recommend to name the function e.g. "ssh_sudo_message" rather then "ssh_message". There are other messages from ssh facility, describing login and logout, for example, which are not covered by this logic.

But this is rather nitpicking, feel free to ignore it if you want to :)